cmd/image/info/info.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 IBM Corp
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package info
+
+import (
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ppc64le-cloud/pvsadm/pkg/utils"
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "info",
+	Short: "Provide info about the image",
+	Long: `Provide info about the image
+pvsadm image info --help for information
+
+
+Examples:
+
+# To get the pvsadm tool version by specifying the image
+pvsadm image info rhcos-46-12152021.ova.gz
+
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		fileName := args[0]
```

Comment:
what happens when user just runs `pvsadm image info`

Reply:
using index like this may result in array out of index error!

Reply:
Done.Added check for the args length.